### PR TITLE
Ccp 1162 pwa 6.x migration

### PR DESCRIPTION
**Pull request by DannyShopgate**
## Description

Updates for PWA 6 migration. i used v6.1.0 branch of the PWA and shop number 31369. Tested with specific items on the category page path Katzen >> Katzennahrung >> Nassfutter. Those items were "Almo Nature HFC Jelly Cat ThunFisch & Garnelen" and "Almo Nature HFC Light Huhn & Aloe 70g" assorted items.

Test for streams and actions are blocked by PWA-1676

## Type of change

- [ ] Bug fix
- [x ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: frontend/subscriptions/index.js [hunk 1]
```diff
@@ -11,14 +12,22 @@ const groupedProductsSubscriptions = (subscribe) => {
...
+      return;
+    }
+    if (hasGroupedProducts(getState(), { productId })) {
+      dispatch(getProductChildren(productId));
     } else {
       dispatch(showAddToCartBar());
     }
   });
+  subscribe(productChildrenReceived$, ({ dispatch }) => {
+    dispatch(hideAddToCartBar());
```
Comment (aaron-martin) [inline frontend/subscriptions/index.js]: Why wait until the child products are received to hide the add to cart bar?  (this is just a question)

File: frontend/portals/ProductHeaderWrapper/index.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const ProductHeaderContainer = connect(({
+  isRendered,
+}) => (
+  <ProductHeader isRendered={isRendered} />
+));
+
+/**
+ * Renders ProductHeader component in product.header.before portal
+ * @param {Object} params RouteContext params
```
[the remark this answers is not in the document]
Comment (DannyShopgate) [reply frontend/portals/ProductHeaderWrapper/index.jsx]: Removed

File: frontend/portals/GroupedProductsWrapper/index.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+const GroupedProductsContainer = connect(({
+  products,
+}) => (
+  <GroupedProducts products={products} />
+));
+
+/**
+ * Renders GroupedProducts component in product.variant-select.before portal
+ * @param {Object} params RouteContext params
```
[the remark this answers is not in the document]
Comment (DannyShopgate) [reply frontend/portals/GroupedProductsWrapper/index.jsx]: Removed

File: frontend/portals/CTAButtonsWrapper/index.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    isAddToCartButtonVisible={isAddToCartButtonVisible}
+    isFavorite={isFavorite}
+    product={product}
+    handleAddToCart={handleAddToCart}
+  />
+));
+
+/**
+ * Renders CTAButtons component in product.ctas portal
+ * @param {Object} params RouteContext params
```
[the remark this answers is not in the document]
Comment (DannyShopgate) [reply frontend/portals/CTAButtonsWrapper/index.jsx]: Removed

**Comment by DannyShopgate** [comment]
Updated zIndex for fav button and push latest changes with updated snapshots

**Comment by aaron-martin** [comment]
@DannyShopgate 
I am still seeing one set of snapshot tests failing. They ar ethe ones in this file. frontend/components/GroupedProducts/components/Product/__snapshots__/spec.jsx.snap

the failing tests are 
* <Product /> › should render as expected when the favorite list is not active
* <Product /> › should render as expected when the favorite list is not active
* <Product /> › should render as expected with regular buttons
* <Product /> › should render as expected with flat buttons

Can you check to make sure you don't have some un-pushed changes in your local repository?

**Pull request merged**